src/frontend/screens/PhotosModal/index.tsx

```diff
@@ -139,6 +136,7 @@ const PhotosModal = ({ navigation }: { navigation: NavigationProp }) => {
         renderTabBar={() => null}
       />
       <ConfirmDeleteModal
+        disableBackdrop={true}
```

Comment:
```suggestion
        disableBackdrop
```

Reply:
Again, stylistic preference, I personally prefer the explicit `true`